docs/user/user/metrics/memory_distribution.md

```diff
@@ -13,6 +13,18 @@ This makes them suitable for measuring memory sizes on a number of different sca
 
 ## Configuration
 
+Memory distributions have a required `memory_unit` parameter, which specifies
+the unit the incoming memory size values are recorded in. The units are the
+power-of-2 units, so "kilobyte" is more correctly a "kibibyte".
+
+```
+- kilobyte == 2^10 ==         1,024 bytes
+- megabyte == 2^20 ==     1,048,576 bytes
+- gigabyte == 2^30 == 1,073,741,824 bytes
+```
+
+Values are automatically converted to and transmitted as bytes.
```

Comment:
If we open the door to documenting how data is stored/transmitted in each metrics docs, we have a lot to do. I think we omit this in favour of leaving it consolidated in the Payload format docs.

Reply:
Sure.  Good point on this one.